chef-utils/spec/spec_helper.rb

```diff
@@ -1,4 +1,4 @@
-require "chef-utils"
+require "chef-utils" unless defined?(ChefUtils)
```

Comment:
this might not work at all.

but i think you may have discovered why i was using `unless defined?(ChefUtils::CANARY)`

Reply:
I updated chefstyle to enforce that